draft-thomson-http-replay.md

```diff
@@ -287,6 +289,37 @@ forwarding the request until the TLS handshake completes, or send a 4NN (Too
 Early) status code in response.  A gateway that is uncertain about whether an
 origin server supports the `Early-Data` header field SHOULD disable early data.
 
+## Consistent Handling of Early Data {#be-consistent}
+
+Consistent treatment of a request that arrives in - or partially in - early data
+is critical to avoiding inappropriate processing of replayed requests.  Requests
+that might be processed prior to the handshake completing MUST be processed the
+same way, even if the handshake is complete when processing occurs.
+
+A TLS implementation MUST ensure that it is possible to correctly identify early
```

Comment:
On the face of it, this seems like it's putting requirements upon TLS implementations. A better way might be to say "To be used for early data in HTTP, a TLS implementation MUST..."